doc/sphinx-guides/source/container/index.rst

```diff
@@ -0,0 +1,26 @@
+Container Guide
+===============
+
+**Contents:**
+
+.. toctree::
+
+  base-image
+
+Running Dataverse software in containers is quite different than in a :doc:`classic installation <../installation/prep>`.
+
+Both approaches have pros and cons. These days, containers are very often used for development and testing,
+but there is an ever rising move for running applications in the cloud using container technology.
+
+**NOTE:**
+**As the Institute for Quantitative Social Sciences (IQSS) at Harvard is running their installations in the classic
+deployment way, the container support is mostly created and maintained by the Dataverse community on a best-effort
+basis.**
+
+This guide is *not* about installation on technology like Docker Swarm, Kubernetes, Rancher or other
```

Comment:
```suggestion
This guide does **not** describe how to run nor build the application with Docker. It only describes how to create a base image for creating application images (executable versions) and how to configure each one. Executable container images are described for example here (`Dataverse on K8s project <https://k8s-docs.gdcc.io>`_).   
```

Reply:
Somehow I can not add a multiline suggestion. The rest of the paragraph can also be removed.

Reply:
You need to select multiple lines first by clicking on the line number where you want to start, hold shift and click on the last line number.

I'll go ahead and resolve this comment and recreate it as multiline.